src/test/java/org/reflections/Issue269.java

```diff
@@ -0,0 +1,53 @@
+package org.reflections;
+
+import java.lang.annotation.*;
```

Comment:
We should explicitly state the class to be used，it is easy to cause name conflicts by this way.